tests/integration/components/o-s-s/alert-test.ts

```diff
@@ -40,7 +40,51 @@ module('Integration | Component | o-s-s/alert', function (hooks) {
     });
   });
 
-  test('it render the extra-content named block', async function (assert) {
+  test('it renders the background-color white in alert when @plain is false', async function (assert) {
+    await render(hbs`<OSS::Alert @plain={{false}} />`);
+
+    assert.dom('.upf-alert .main-container').hasNoClass('main-container--plain');
+  });
+
+  test('it renders the background-color grey in alert when @plain is true', async function (assert) {
+    await render(hbs`<OSS::Alert @plain={{true}} />`);
+
+    assert.dom('.upf-alert .main-container').hasClass('main-container--plain');
+  });
+
+  test('it renders the background-color grey in alert when @plain is undefined', async function (assert) {
+    await render(hbs`<OSS::Alert />`);
+
+    assert.dom('.upf-alert .main-container').hasClass('main-container--plain');
+  });
```

Comment:
These are the same type of tests, they can be grouped into a module for better readability :
```typescript
module('@plain parameter', () => {}
   test('If true, the background-color is grey', async function (assert) {});
   test('If false, the background-color is white', async function (assert) {});
   test('If undefined, the background-color is grey', async function (assert) {});
);
```